DashWallet/Sources/Models/CrowdNode/TxFilters/CoinsToAddressTxFilter.swift

```diff
@@ -30,24 +30,33 @@ public class CoinsToAddressTxFilter: TransactionFilter {
 
     func matches(tx: DSTransaction) -> Bool {
         fromAddresses.removeAll()
-        tx.inputAddresses.forEach {
-            if let address = $0 as? String {
-                fromAddresses.insert(address)
-            }
-        }
 
         // TODO: if CrowdNode inputs aren't from our own transaction, the fee might not be present.
         // Need another way to detect an error response in this case.
-        let feeUsed = tx.feeUsed
```

Comment:
This was the biggest performance offender since calculating the fee every time is a stupidly expensive operation.